### DISCUSSION TITLE
home

### DISCUSSION THREAD

**Discussion by giscus** [Snailclimb/JavaGuide#1655] [Announcements]
# home

Java学习&&面试指南

https://javaguide.cn/home.html

**Comment by Snailclimb** [comment]
用心做内容，持续完善，加油！！！

**Reply by demo-zexuan** [reply]
加油💪🏻

**Reply by itwanger** [reply]
赶紧来前排占个坑！！！！！G 哥就是我的领路人，爱了爱了。

**Reply by mynameismaoyu** [reply]
数据库那一章的mysql高性能优化规范建议的数据库sql开发规范的第2条 举例的sql是不是有问题 这个sql并不会导致索引失效

**Reply by Snailclimb** [reply]
> 数据库那一章的mysql高性能优化规范建议的数据库sql开发规范的第2条 举例的sql是不是有问题 这个sql并不会导致索引失效

嗯嗯 那个没有说清楚，可以看看这篇文章：[MySQL中的隐式转换造成的索引失效](https://javaguide.cn/database/mysql/index-invalidation-caused-by-implicit-conversion.html) 。

**Comment by Guo-mingming** [comment]
前排抢个座！秋招冲冲冲！

**Reply by Snailclimb** [reply]
> 前排抢个座！秋招冲冲冲！

祝找到满意的 offer!冲冲冲！加油！

**Comment by DevilFeel** [comment]
抢个前排

**Comment by linzhuohui** [comment]
前排，Guide哥牛逼

**Comment by luogege123** [comment]
我计算机大猛子ym来抢前排啦！我不要做小牛马，我要做牛马领导！
太狠了兄弟萌，互联网大厂冲冲冲！

**Comment by shilohooo** [comment]
古德古德

**Comment by demo-zexuan** [comment]
好耶，听说可以评论了，火速来试试！

**Comment by yuyueq** [comment]
听说可以评论啦，占个楼
